packages/node_modules/samples/browser-plugin-meetings/test/wdio/spec/mediatest.js

```diff
@@ -57,15 +56,15 @@ describe('samples', () => {
       });
 
       it('places call from spock to mccoy', () => {
-        browserSpock.setValue('#invitee', spock.emailAddress);
+        browserSpock.setValue('#invitee', mccoy.emailAddress);
```

Comment:
@taymoork2 This should fix the issue. Spock was always calling Spock before, so McCoy never got a notification.